packages/core/src/command/hl7v2-subscriptions/__tests__/hl7v2-roster-generator.test.ts

```
@@ -133,79 +190,24 @@
       delete patient.data.contact;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)